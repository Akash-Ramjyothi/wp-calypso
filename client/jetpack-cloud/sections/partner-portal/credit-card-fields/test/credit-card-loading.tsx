/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/react';
import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import CreditCardLoading from '../credit-card-loading';

describe( '<CreditCardLoading>', () => {
	test( 'testing', () => {
		const initialState = {};
		const mockStore = configureStore();
		const store = mockStore( initialState );

		render(
			<Provider store={ store }>
				<CreditCardLoading />
			</Provider>
		);

		expect( document.body ).toMatchSnapshot();
	} );
} );
