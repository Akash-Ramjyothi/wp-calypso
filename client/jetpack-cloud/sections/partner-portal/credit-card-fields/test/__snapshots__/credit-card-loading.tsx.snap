// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreditCardLoading> testing 1`] = `
<body>
  <div>
    <div
      class="credit-card-fields credit-card-fields--placeholder"
    >
      <div
        class="credit-card-fields__form"
      >
        <div
          class="credit-card-fields__field"
        />
        <div
          class="credit-card-fields__field"
        />
        <div
          class="credit-card-fields__field"
        />
      </div>
    </div>
  </div>
</body>
`;
